backend/endpoints/session/routes.js

```diff
@@ -1,2 +1,10 @@
 export const location = '/api/session';
-export const routes = {};
+export const routes = {
+  get: {
```

Comment:
I don't think we need this route at all right now, but there is no reason not to include it right now IMO.

Can you add a comment saying this is for the beta staging release if we want to remove it later.